backend/src/test/java/ai/verta/modeldb/FindDatasetEntitiesTest.java

```diff
@@ -656,14 +656,10 @@ public void findDatasetsByMetricsAndTagsTest() {
 
   /** Find dataset with value of endTime */
   @Test
-  @Ignore
-  public void findDatasetsByDatasetEndTimeTest() {
-    LOGGER.info("FindDatasets By Dataset EndTime test start................................");
-
-    DatasetTest datasetTest = new DatasetTest();
+  public void findDatasetsByDatasetTimeUpdatedTest() {
+    LOGGER.info("FindDatasets By Dataset TimeUpdated test start................................");
 
-    Value stringValue =
-        Value.newBuilder().setStringValue(String.valueOf(dataset4.getTimeUpdated())).build();
+    Value stringValue = Value.newBuilder().setNumberValue(dataset4.getTimeUpdated()).build();
```

Comment:
please add PR descriptions.
 probably change the var name to numberValue ?

in the PR description please mention when did we start breaking the test , and why do we end up updating the test instead of making code change.

Reply:
TimeUpdated is not a string but backend supports the string as a value and finds endpoints work fine. but I have just updated it correctly.

Reply:
Fix variable name